README.md

```diff
@@ -30,14 +30,52 @@ if you want to contribute to the port please come to the `mathlib4` stream.
   find Mathlib -name "*.lean" | env LC_ALL=C sort | sed 's/\.lean//;s,/,.,g;s/^/import /' > Mathlib.lean
   ```
 
-## Building HTML documentation
+### Downloading cached build files
+
+Run `lake exe cache get` to download cached build files that are computed by `mathlib4`'s automated workflow.
+If `tar` terminates with an error, it means that you might have ended up with corrupted files.
+In this case, run `lake exe cache get!` to overwrite them (`get` won't try to download the same file again).
+
+Call `lake exe cache` to see its help menu.
+
+### Building HTML documentation
 Building HTML documentation locally is straightforward:
 ```
 lake -Kdoc=on build Mathlib:docs
 ```
 The HTML files can then be found in `build/doc`.
 
-## Dependencies
+### Dependencies
 If you want to update dependencies, use `lake update -Kdoc=on`.
 This will update the `lean_packages/manifest.json` file correctly.
 You will need to make a PR after committing the changes to this file.
+
+## Using `mathlib4` as a dependency
+
+If you want to start a project that uses `mathlib4` as a dependency, you can run:
+```
+lake init MyProject math
+```
+
+Important: the command above requires a more recent toolchain set as default, which can be done with, for example:
+```
+elan default leanprover/lean4:nightly-2023-01-29
+```
+
+Or, if you already have a project and you want to be able to use `mathlib4`, add these lines to your `lakefile.lean`:
+```
+require std from git
```

Comment:
Should this be `std`? (I don't know what I'm talking about, but it looks like this should be
```suggestion
require mathlib4 from git
```